roles/oneagent/tests/ansible/config.py

```diff
@@ -1,40 +1,40 @@
 import shutil
 from typing import Any
 
-from ansible.constants import (
+from constants import (
     ANSIBLE_CONNECTION_KEY,
     ANSIBLE_PASS_KEY,
-    ANSIBLE_RESOURCE_DIR,
+    ANSIBLE_RESOURCES_PATH,
```

Comment:
```suggestion
    ANSIBLE_RESOURCES_DIR_PATH,
```
Personally, I appreciate having additional information on whether a path is a directory. It makes reading the code easier. I see that you sometimes use it, for example, `WORK_DIR_PATH`.